backend/generating/AutomarkableTask.java

```diff
@@ -0,0 +1,18 @@
+package generation;
+
+import java.util.List;
+
+public class AutomarkableTask extends Task {
+    private String language;
+    private String url;
+
+    public AutomarkableTask(String language, String url, String taskText, boolean isAutomarkable, List<Task> subTaskList, boolean isSubTask) {

```

Comment:
Why do we have an isAutomarkable field in the constructor for an Automarkable task?

Couldn't we just override the `isAutomarkable` method in the superclass and return true?